packages/accounts/accounts-tab/client.js

```diff
@@ -4,23 +4,12 @@ ReactiveTemplates.onRendered('accounts.index', function() {
 })
 
 ReactiveTemplates.helpers('accounts.index', {
-  users: function () {
-    return Meteor.users.find({}, { sort: { createdAt: -1 } });
-  },
-  buttons: function() {
-    var self = this;
-    return _.filter(orion.accounts._adminUsersButtons, function(value, key, list){
-      if (typeof value.shouldShow != 'function') {
```

Comment:
You deleted privileges for buttons display? Are you handling this elsewhere in a different manner?